### ISSUE TITLE
Scrolling in left pane doe not work when frozen columns in combination with grouping

### ISSUE THREAD

**Issue by celsius65** [6pac/SlickGrid#525]
When I setup some frozen columns in combination with groupheaders then the scrolling does not work when the mouse is in the left pane.

If I only use frozen columns then I have no problem. When I use only grouping then evertything works also fine. I have tried to figure out what is going on but no luck. 
The easiest way to check is to take the grouping column example en freeze a few columns.

Thanks,
Kelvin

**Comment by ghiscoding** [comment]
Vertical scrolling is disabled on the left container when freezing columns is enabled. It's by design and only right container will be scrollable.

**Comment by celsius65** [comment]
Well, that's what I thought also.

I then checked the "example-frozen-columns.html" example and there the scrolling on the left pane is enabled. 
So I thought it might be bug. 

Do you have any idea why it could be working in that particular example?

Thanks,
Kelvin

**Comment by ghiscoding** [comment]
@celsius65 
I'm not really sure what you mean, the [example-frozen-columns.html](https://6pac.github.io/SlickGrid/examples/example-frozen-columns.html) works as intended (scrolling only available on the right side) as demoed below

![chX22HGNJF](https://user-images.githubusercontent.com/643976/88727882-83d0ef00-d0fe-11ea-9aa2-38bbcbbccc69.gif)

**Comment by celsius65** [comment]
Oh we are really speaking a different language :-), I'm sorry for not making myself clear and thank for the trouble to make that gif. 

To take your gif as an example, what I meant was the following:
When my mousepointer is on the left part of the grid, i.e. on the frozen column section, the grid is not responding to the mousewheel thus, no vertical scrolling.
If I move the pointer to the right part of the grid reacts on the mousewheel as expected. 

It is just the mousewheel I have troubles with. Everything else works excellent!

Did I make myself clear with above explanation? If not, don't hesitate to ask.

Thanks,
Kelvin

**Comment by ghiscoding** [comment]
Ok that is totally different, but even then the mouse wheel works for me on both side, but that is a computer mouse not a touchpad. But if you have some issues, you have to know that the Frozen Grid feature is actually built using 2 different DOM container elements (left/right) and because of that there are some adjustment that are required for everything to work... for example, you will see that a background color applied with just CSS won't show up on the entire row (if you're on the left side, then only the row on the left side will have the background color) and that can be fix by JS/jQuery code as such

```js
    // with frozen (pinned) grid, in order to see the entire row being highlighted when hovering
    // we need to do some extra tricks (that is because frozen grids use 2 separate div containers)
    // the trick is to use row selection to highlight when hovering current row and remove selection once we're not
    grid.onMouseEnter.subscribe(event => {
      const cell = grid.getCellFromEvent(event);
      grid.setSelectedRows([cell.row]); // highlight current row
      event.preventDefault();
    });
    grid.onMouseLeave.subscribe(event => {
      grid.setSelectedRows([]); // remove highlight
      event.preventDefault();
    });
```
basically what I'm trying to say is that you might need to add such JS/jQuery code to fix some of the wanted behaviors.

... and after some research, there was that PR #327 which we applied some code change for the scrolling, it should have fixed this issue (which it did for my perspective since I can't replicate what you're saying)
